content/docs/command-reference/checkout.md

```diff
@@ -207,8 +204,30 @@ $ md5 model.pkl
 MD5 (model.pkl) = 98af33933679a75c2a51b953d3ab50aa
 ```
 
-What happened is that DVC went through the DVC-files and adjusted the current
-set of <abbr>output</abbr> files to match the `outs` in them.
+DVC went through `dvc.lock` and adjusted the current set of <abbr>outputs</abbr>
+to match the `outs` in it.
+
+## Example: Specific targets
+
+`dvc checkout` only affects the tracked data corresponding to any given
+`targets`:
+
+```dvc
+$ git checkout master
+$ dvc checkout            # Start with latest version of everything.
+
+$ git checkout baseline-experiment -- dvc.lock
+$ dvc checkout model.pkl  # Get previous model file only.
+```
+
+Note that `dvc checkout` support granularity for files found in tracked
```

Comment:
support -> supports

Reply:
can we simplify this? `dvc checkout` by default checks out everything, but it can accept any path to a DVC-tracked files (including path to a file inside DVC-tracked directory) or a stage name. In this case it checks out only files or directories related to this target(s).

Reply:
Simplified in 9879c1de.